### PR TITLE
My rental improvements

### PR DESCRIPTION
**Pull request by jfnumainville**
Hi @andking, @Suebxl  and @MarkSoosaipillai , can one of you please approve?

Here are the changes:

1- Seed: Improve file so that each mascot get  a distinct address and that users list  used for creating costumes and reservations exclude me (for the demo)
2- My rentals page: Added the "Owner" and "Requested BY" fields to the tabs.   Added margin between rows and centered content.

image

image

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -36,8 +36,8 @@
 # end
 
 
-#Retrieving the list of all user ids
-p user_ids_list = User.all.pluck(:id)
+#Retrieving the list of all user ids, excluded Mark from the list as we will be using his ID for the demo and we do not want him to own costumes or to have reservations before the demo.
+p user_ids_list = User.all.where.not(name:"Jean-Francois").pluck(:id)
```
Comment (andking) [inline db/seeds.rb:40]: Mark or Jean-Francois?

Comment (jfnumainville) [reply db/seeds.rb:40]: Hey @andking , comment should say Jean-Francois, will update!

Comment (jfnumainville) [reply db/seeds.rb:40]: Hey @andking , update is pushed

**Review by Suebxl — APPROVED** [review]

**Comment by Suebxl** [comment]
Nice improvements.  Go Go tonight demo presentation of the Airbnb project

**Pull request merged**